depmapomics/release.py

```diff
@@ -0,0 +1,545 @@
+#release.py
+import os
+import yaml
+from genepy.utils import helper as h
+import pandas as pd
+from depmapomics.config import *
+from gsheets import Sheets
+from taigapy import TaigaClient
+tc = TaigaClient()
+
+virtual = {}
+INFO = {}
+
+def clean():
+  global virtual
+  global INFO
+  virtual = {}
+  INFO = {}
+
+
+def run(release=RELEASE, notebooks=RUN_NOTEBOOKS):
+  cmd = ''
+  for val in notebooks:
```

Comment:
no, its just a python function. it can be run from anywhere. 

It does not work yet because within notebooks there is a step which need manual intervention until we add Emeril

Reply:
Gotcha. What's the eventual goal of this function?